### ISSUE TITLE
Error: downloadSecureFile is not a function

### ISSUE THREAD

**Issue by dustinwoodhouse** [jabbera/vsts-authenticode#24]
Getting the following run-time error when signing my code. I've set the task to consume a .pfx file that is stored in Azure Devops as a seure file.

[section]Starting: Authenticode Sign
==============================================================================
Task         : Authenticode Sign
Description  : Authenticode sign executable code.
Version      : 4.0.0
Author       : Michael Barry
Help         : 
==============================================================================
[error]Unable to sign TypeError: this.serverConnection.getTaskAgentApi(...).downloadSecureFile is not a function
(node:4360) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: this.serverConnection.getTaskAgentApi(...).downloadSecureFile is not a function
[section]Finishing: Authenticode Sign

**Comment by dustinwoodhouse** [comment]
I'm using v4.x

**Comment by mikehachen** [comment]
Same problem hiere. When I switch back to version 3 everything works.

**Comment by rvanmaanen** [comment]
@jabbera It would be awesome if the PR that fixes this issue can be looked at soon. Version 3 isn't available with a new install, so this bug is blocking anyone trying the extension.

**Comment by jabbera** [comment]
I’ll merge it today. Cheers!

**Comment by jabbera** [comment]
The PR didn't seem to work. The task is terminating after the file is downloaded for some reason. Will look into it more tomorrow. However I'm thinking about deprecating this functionality as microsoft has a built in task to support this. Is there a reason you aren't using it? (https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-secure-file?view=azure-devops)

```
2019-12-26T21:00:01.3195759Z ##[debug]Agent.TempDirectory=D:\a\_temp
2019-12-26T21:00:01.3198512Z ##[debug]Absolute path for pathSegments: D:\a\_temp,A.pfx = D:\a\_temp\A.pfx
2019-12-26T21:00:01.3198675Z ##[debug]Downloading secure file contents to: D:\a\_temp\A.pfx
2019-12-26T21:00:01.5880322Z ##[debug]SYSTEM.TEAMPROJECT=vsts-authenticode
2019-12-26T21:00:01.5880842Z ##[debug]secure file ticket for id f3c76c92-a70a-43f0-9e0a-b930ecbb9121 = ***
2019-12-26T21:00:01.8010952Z ##[section]Finishing: Authenticode Sign
```

**Comment by jabbera** [comment]
All set. Released in 5.0.4
